src/Utils/removeDuplicates.ts

```diff
@@ -0,0 +1,81 @@
+export const groupAndMergeByColumn = (
```

Comment:
use proper namings. nobody will know this is a csv utility function without context.

```suggestion
export const csvGroupByColumn = (
```